templates/prepare.j2

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
I want us to collectively promise this is MVP and will be coverted to Python at some point

Reply:
Sure it could be, but this isn't just custom scripts I created these are the gitlab provided example scripts for custom lxd. If there were an issue with the custom executor using their scripts might be preferable?

Although as long as pylxd accomplishes the same goal I guess no harm. I wouldn't mind having it all in Python.